### PR TITLE
Parameters

### PR DESCRIPTION
**Pull request by kba**

### CODE REVIEW

File: site/en/user_guide.md [hunk 1]
```diff
@@ -198,23 +198,23 @@ ways make use of the following syntax:
 ```sh
 -I Input-Group    # folder of the files to be processed
 -O Output-Group   # folder for the output of your processor
--p parameter.json # indication of parameters for a particular processor
+-P parameter      # indication of parameters for a particular processor
```
Comment (kba) [inline site/en/user_guide.md:201]: `-P KEY VALUE` ?

File: site/en/user_guide.md [hunk 2]
```diff
@@ -198,23 +198,23 @@ ways make use of the following syntax:
...
 ```
 
 For some processors parameters are purely optional, other processors as e.g. `ocrd-tesserocr-recognize` won't work without one or several parameters.
 
 ### Calling a single processor
 If you just want to call a single processor, e.g. for testing purposes, you can go into your workspace and use the following command:
 ```sh
-ocrd-[processor needed] -I [Input-Group] -O [Output-Group] -p [path to parameter.json]
+ocrd-[processor needed] -I [Input-Group] -O [Output-Group] -P [param value]
```
Comment (kba) [inline site/en/user_guide.md:209]: `-P [PARAM_NAME PARAM_VALUE]`?

File: site/en/user_guide.md [hunk 3]
```diff
@@ -227,25 +227,12 @@ ocrd-anybaseocr-crop  -I OCR-D-IMG -O OCR-D-BIN,OCR-D-IMG-BIN
...
-
-Instead of calling a `parameter.json` file you can also directly
-write down the parameters when invoking a processor:
-
-```sh
--p '{"[parameter]": "[value]"}'
-```
-
-**Note:** If multiple parameters are necessary they have to be separated by a comma. (No comma after the last parameter!)
+**Note:** If multiple parameters are necessary, each of them has to be preceded by `-P`
```
Comment (kba) [inline site/en/user_guide.md:230]: **Note:** If multiple parameter key-value pairs are necessary, each of them has to be preceded by `-P`

File: site/en/user_guide.md [hunk 4]
```diff
@@ -227,25 +227,12 @@ ocrd-anybaseocr-crop  -I OCR-D-IMG -O OCR-D-BIN,OCR-D-IMG-BIN
...
 
 E.g.: 
 
 ```sh
--p '{"[param1]": "[value1]", "[param2]": "[value2]", "[param3]": "[value3]"}'
+-P param1 value1 -P param2 value2] -P param3 value3
```
Comment (kba) [inline site/en/user_guide.md:235]: much better.

Comment (kba) [reply site/en/user_guide.md:235]: but there's a spurious `]` isn't it?

File: site/en/workflows.md [hunk 5]
```diff
@@ -64,7 +64,7 @@ the raw image is taken and enhanced by e.g. grayscale conversion, brightness nor
 	  for parameters and command examples (presets) see [the Readme](https://github.com/bertsky/ocrd_wrap#ocr-d-processor-interface-ocrd-preprocess-image)
 	  </td>
       <td><code>
-	  ocrd-preprocess-image -I OCR-D-IMG -O OCR-D-PREP -p '{"output_feature_added": "binarized","command": "scribo-cli sauvola-ms-split '@INFILE' '@OUTFILE' --enable-negate-output"}'
+	  ocrd-preprocess-image -I OCR-D-IMG -O OCR-D-PREP -P output_feature_added binarized -P command "scribo-cli sauvola-ms-split @INFILE @OUTFILE --enable-negate-output"
```
Comment (kba) [inline site/en/workflows.md:67]: You tested this, right? We don't need quotes around `@INFILE`/@OUTFILE`?

File: site/en/workflows.md [hunk 6]
```diff
@@ -962,9 +968,9 @@ In this processing step, the recognized text is corrected by statistical error m
   <tbody>
     <tr>
       <td>ocrd-cor-asv-ann-process</td>
-      <td><code>{“textequiv_level”:”line”,”model_file”:”/path/to/model/model.h5”}</code></td>
+      <td><code>{“textequiv_level”:”word”,”model_file”:”/path/to/model/model.h5”}</code></td>
```
Comment (kba) [inline site/en/workflows.md:971]: these are typographic (`“`) quotes, they should be regular double quotes `"`

File: site/en/user_guide.md (hunk 1 above)
Comment (EEngl52) [reply site/en/user_guide.md:201]: At first I wanted to write key value, but everywhere else it already said param value. rather change or leave it?

File: site/en/workflows.md (hunk 5 above)
Comment (EEngl52) [reply site/en/workflows.md:67]: yes, it works like this. also works with single quotes though

File: site/en/user_guide.md (hunk 1 above)
Comment (kba) [reply site/en/user_guide.md:201]: Maybe add an introductory note:

> **NOTE:** The `-P` option accepts a parameter name and a parameter value. When we write `-P parameter`, we mean that `parameter` consists of `parameter name` and `parameter value`.
?

Comment (EEngl52) [reply site/en/user_guide.md:201]: sounds good to me :+1:

**Pull request merged**